lib/sinon/stub.js

```diff
@@ -164,8 +164,6 @@ var proto = {
         });
     },
 
-    resetHistory: spy.resetHistory,
```

Comment:
This is now inherited from `proxy.js`, the `spy` module does not expose `resetHistory` anymore, so this would assign `undefined`.